tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -320,6 +320,36 @@ public function testBooleanDefault(callable $comparatorFactory): void
         self::assertFalse($diff);
     }
 
+    /**
+     * @param callable(AbstractSchemaManager):Comparator $comparatorFactory
+     *
+     * @dataProvider \Doctrine\DBAL\Tests\Functional\Schema\ComparatorTestUtils::comparatorProvider
+     */
+    public function testGeneratedColumn(callable $comparatorFactory): void
+    {
+        $wrappedConnection = $this->connection->getWrappedConnection();
+        assert($wrappedConnection instanceof ServerInfoAwareConnection);
+        if (version_compare($wrappedConnection->getServerVersion(), '12.0', '<')) {
```

Comment:
Now that you have a dedicated platform for version 12, you can check the type of `$this->connection->getDatabasePlatform()` instead.

Reply:
changed